generators/app/index.js

```diff
@@ -43,6 +43,11 @@ module.exports = class extends Generator {
       OpenUI5: "1.90.1", //"1.60.0",
       SAPUI5: "1.90.0" //"1.77.0"
     };
+    
```

Comment:
Also mixed indentation. At least in GitHub selecting the whitespace in front of the unchanged lines shows they are spaces, while in front of the new lines entire tabs are selected.